pkg/operator/api/api.go

```diff
@@ -219,34 +219,41 @@ func (a *apiServer) ComponentUpdate(in *operatorv1pb.ComponentUpdateRequest, srv
 		a.connLock.Unlock()
 	}()
 	chWrapper := initChanGracefully(updateChan)
+	updateComponentFunc := func(c *componentsapi.Component) {
+		err := processComponentSecrets(c, in.Namespace, a.Client)
+		if err != nil {
+			log.Warnf("error processing component %s secrets: %s", c.Name, err)
+			return
+		}
 
-	for c := range updateChan {
-		go func(c *componentsapi.Component) {
-			err := processComponentSecrets(c, in.Namespace, a.Client)
-			if err != nil {
-				log.Warnf("error processing component %s secrets: %s", c.Name, err)
-				return
-			}
-
-			b, err := json.Marshal(&c)
-			if err != nil {
-				log.Warnf("error serializing component %s (%s): %s", c.GetName(), c.Spec.Type, err)
-				return
+		b, err := json.Marshal(&c)
+		if err != nil {
+			log.Warnf("error serializing component %s (%s): %s", c.GetName(), c.Spec.Type, err)
+			return
+		}
+		err = srv.Send(&operatorv1pb.ComponentUpdateEvent{
+			Component: b,
+		})
+		if err != nil {
+			log.Warnf("error updating sidecar with component %s (%s): %s", c.GetName(), c.Spec.Type, err)
```

Comment:
I think debug level is enough to ensure useful logs are not overwhelmed.

Reply:
Ok, i change it at right.  

At present, our dapr ecological library uses log levels. Are there any specifications? I can have a unified look